### DISCUSSION TITLE
How do I get this working under `root`?

### DISCUSSION THREAD

**Discussion by stevenxxiu** [3rd/image.nvim#134] [Q&A]
Thanks for the plugin.

I'm using *Arch Linux*, *WezTerm*, the *Kitty* protocol.

This works fine for my regular user, and even over SSH through *Tmux*, but only shows blank spaces under `root`.

I tried going through some of the code:

- In `image.nvim/lua/image/renderer.lua`, the image did get rendered and resized properly to `/run/user/0/nvim.root/I2uH2Z/0/MTAwMDoxOjA6L3RtcC9mb28ucG5n-resized-120-119.png`.
- In `image.nvim/lua/image/backends/kitty/helpers.lua` `write_graphics()`, there are things being written.

Wondering if it's a *WezTerm* issue, I ran the script <https://github.com/wez/wezterm/blob/20240203-110809-5046fc22/test-data/kitty-png.py> with my file:

    # python /tmp/kitty-png.py /tmp/foo.png

It shows the image no problems.

Any way I can debug this further?

**Comment by 3rd** [comment]
No clue, but it's probably a sign from the gods not to run it under root :(
